documentation/asciidoc/topics/proc_securing_cross_site_connections.adoc

```diff
@@ -2,7 +2,10 @@
 = Securing cross-site connections
 
 [role="_abstract"]
-Add keystores, and optional trust stores, so that {brandname} clusters can encrypt RELAY messages and secure cross-site replication traffic.
+Add keystores and trust stores so that {brandname} clusters can encrypt RELAY messages and secure cross-site replication traffic.
+
+Securing cross-site connections is optional if you use a `NodePort` or `LoadBalancer` as the expose type.
+If you use an {openshiftshort} `Route` as the expose type for cross-site replication, you must add a keystore and configure encryption for RELAY messages.
```

Comment:
@pruivo Can we add a line to explain why TLS is necessary with a Route. The [con_cross_site_expose_types.adoc](https://github.com/infinispan/infinispan-operator/pull/1488/files#diff-d7ac011fd53604870d64ecd94d49e9dcd7a56ab1bf22741d88574e186e7e9d56) docs implies it's because we use passthrough TLS, but why can't we configure the Route without TLS?

Reply:
AFAIK, it is not supported [1]

```
The methods are recommended, in order or preference:
1 If you have HTTP/HTTPS, use an Ingress Controller.
2 If you have a TLS-encrypted protocol other than HTTPS. For example, for TLS with the SNI header, use an Ingress Controller.
3 Otherwise, use a Load Balancer, an External IP, or a NodePort.
```
We are using TLS+SNI to get the JGroups traffic through the Route. 

https://docs.openshift.com/container-platform/4.10/networking/configuring_ingress_cluster_traffic/overview-traffic.html